File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int pyramid_height;
void pyramid_generator(void);

int main(void)
{
    do
    {
        pyramid_height = get_int("How high is the pyramid? ");
    }
    while (pyramid_height <= 0 || pyramid_height > 8);
    pyramid_generator();

}

void pyramid_generator()
{
    //Pyramid height
    for (int j = 0; j < pyramid_height; j++)
    {
        //print left spaces
        for (int k = pyramid_height - 1; k > j; k--)
        {
            printf(" ");
        }

        //print left pyramid
        for (int l = 0; l <= j; l++)
        {
            printf("#");
        }

        //print gap
        printf("  ");

        //print right pyramid
        for (int x = 0; x <= j; x++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

while True:
    pyramid_height = get_int("How High is the pyramid ")
    if pyramid_height > 0 and pyramid_height < 9:
        break


for x in range(pyramid_height):

    #print left spaces
    spaces = pyramid_height - 1
    while spaces > x:
        print(" ", end="")
        spaces -=1

    #print left pyramid
    left_pyramid = 0
    while left_pyramid <= x:
        print("#", end="")
        left_pyramid += 1

    #print gap
    print("  ", end="")

    #print right pyramid
    right_pyramid = 0
    while right_pyramid <= x:
        print("#", end="")
        right_pyramid += 1

    #print Next Line
    print("")